tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -0,0 +1,20 @@
+package globalrolesv2
+
+import (
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	roleOwner      = "cluster-owner"
+	standardUser   = "user"
+	localcluster   = "local"
+	crtbOwnerLabel = "authz.management.cattle.io/grb-owner"
+)
+
+var globalRole = v3.GlobalRole{
```

Comment:
Can we modify this to include random name generation for the global role, given that we create a global role in every test case? If so, we can remove this line from the test case: https://github.com/rancher/rancher/pull/42933/commits/4659d21c1e57fcacb60248175ff04f4c707f40e1#diff-40431274d5a7316127ba57ab2cb9388b7b73aced3fe557ad7eab5db612752127R63
var globalRole = v3.GlobalRole{
    ObjectMeta: metav1.ObjectMeta{
        Name: namegen.AppendRandomString("testgr-"),
    },
    InheritedClusterRoles: []string{},
}